packages/build/webpack/webpack.dev.config.js

```diff
@@ -29,12 +29,16 @@ module.exports = {
     filename: '[name].js',
     chunkFilename: '[name].js',
   },
-  devtool: false,
+  devtool: 'source-map',
   mode: 'development',
-  plugins: [configFactory.plugins.reactRefresh()],
+  plugins: [
+    configFactory.plugins.tsChecker(),
```

Comment:
The problems with the storybook got me thinking: what's the advantage of type checking code within webpack rather than as a separate step before building?

The only thing that comes to my mind is that the dev server will show you a nice error but is there anything else that this gives us?

Reply:
Type checking is a lot faster when you can run a TypeScript compiler continuously instead of from fresh every time. I much prefer the development flow of having Webpack type check instead of having to run another command to do so, or rely on my IDE picking up changes. I left the `yarn type-check` script in for those who still want to use the command.